src/constants.ts

```diff
@@ -92,16 +92,19 @@ export enum ScmType {
 
 export enum CodeAction {
     Deploy = 'Deploy',
-    Debug = 'Debug'
+    Debug = 'Debug',
 }
 
+/**
+ * Default host connection keys
+ */
 export enum ConnectionKey {
     Storage = 'AzureWebJobsStorage',
     StorageIdentity = 'AzureWebJobsStorage__accountName',
     EventHubs = 'EventHubsConnection',
     DTS = 'DURABLE_TASK_SCHEDULER_CONNECTION_STRING',
-    DTSHub = 'TASKHUB_NAME',
-    SQL = 'SQLDB_Connection'
+    DTSHub = '%TASKHUB_NAME%',
```

Comment:
Are these %'s intentional?

Reply:
Yeah, I was previously wrapping the key itself, but decided we should just include them here was the "default" connection key string for the host.json